components/chart/description/index.tsx

```diff
@@ -1,13 +1,16 @@
 import { Offcanvas } from 'react-bootstrap';
-import { BsX } from 'react-icons/bs';
-import React, { PropsWithChildren, useEffect, useRef } from 'react';
+import { BsX } from "react-icons/bs";
+import { FaLink } from "react-icons/fa";
+import React, { PropsWithChildren, useEffect, useRef, useState } from 'react';
+import { stringify, parse } from 'query-string';
 import styled from 'styled-components';
 import { NodeType } from '../types';
 
 type DrawerProps = {
-  module: NodeType;
-  open: boolean;
-  onClose: () => void;
+    module: NodeType;
+    open: boolean;
+    onClose: () => void;
+    children: any;
```

Comment:
While we are using PropsWithChildren no need to define children here, and let's try to avoid using any type as possible